phpBB/phpbb/search/backend/fulltext_sphinx.php

```diff
@@ -631,7 +631,7 @@ public function tidy(): void
 	 */
 	public function create_index(int &$post_counter = 0): ?array
 	{
-		if ($this->index_created())
```

Comment:
I am not sure why I did this.

Reply:
Probably the truncate table lines can be removed

Reply:
Yeah, truncating a newly created table sounds kind of wrong.